MN/Algoritmi/Valori-Proprii/inverse.m

```diff
@@ -0,0 +1,14 @@
+function [Ai] = inverse(A, miu, I)
```

Comment:
trebuie stearsa functia asta cu totul

Reply:
Te referi sa fac introduc inversarea in interioriul scriptului mare? E ok prin metoda asta? Nu inteleg exact la ce te referi

Reply:
tu nu trebuie sa calculezi inversa, ci sa rezolvi sistemul (A - miu * I) * z = y.
Pe asta il rezolvi cu ceva deja existent in repo, la alegere. (LU, QR, Gauss, ce vrei tu)
E fix un sistem de forma A * x = b. A = (A - miu * I), x = z (pe care il afli), b = y.